views/verify_email.ejs

```diff
@@ -82,7 +82,7 @@
         <p class="email-message">
           You are welcome to Aphia! We are glad to have you onboard. Kindly
           click on this link to verify your account
-          <a href="https://aphia.onrender.com/verify/?email=<%=email %>">Click here</a>
+          <a href="https://aphia.onrender.com/api/verify/?email=<%=email %>">Click here</a>
```

Comment:
Make changes to this to reflect the front end URL unless you have a get method that handles this route

Reply:
we need to test it from the backend first